.github/workflows/maven.yml

```diff
@@ -31,6 +31,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           submodules: 'true'
+          fetch-depth: 0
```

Comment:
https://github.com/actions/checkout/blob/main/README.md#:~:text=triggered%20the%20workflow.-,Set%20fetch%2Ddepth%3A%200,-to%20fetch%20all